setup.cfg

```diff
@@ -3,5 +3,6 @@
 universal=1
 
 [bdist_rpm]
+# Due to changes in Python's ConfigParser, this only works in Python 3. For Python 2, remove one %
+release=1%%{?dist}
```

Comment:
for the life of me I could not find a way to get this to work for both Py2 and Py3.  So I decided to go for Py3 and we'll worry about it later if anyone ever wants Py2 support